packages/main/src/ColorPicker.ts

```diff
@@ -91,11 +91,11 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	 * Defines the currently selected color of the component.
 	 *
 	 * **Note**: use HEX, RGB, RGBA, HSV formats or a CSS color name when modifying this property.
-	 * @default "rgba(255, 255, 255, 1)"
+	 * @default ""
```

Comment:
I would return this too. If we compare this component to the native `input[type=color]` there is always a value

Reply:
I agree. The reason why it was removed is because in openui5 we have it with no default value. But as we want to be based on native components, I will return the default value.